binr/radiff2/radiff2.c

```diff
@@ -91,7 +91,7 @@ static void readstr(char *s, int sz, const ut8 *buf, int len) {
 		return;
 	}
 	s[sz - 1] = 0;
-	while (*s && *s == '\n') {
+	while (*s && *s != '\n') {
```

Comment:
this condition looks wrong to me, because it will skip the word in s, when what we want is to skip the newlines

Reply:
ah okay, than it can stay as is